### ISSUE TITLE
Are other presets allowed?

### ISSUE THREAD

**Issue by airtonix** [jscutlery/semver#560]
https://github.com/jscutlery/semver/blob/9997fcabca2a01a72ea351033b2bf91efcd1f83b/packages/semver/src/executors/version/index.ts#L230-L233

This seems to indicate that only angular or conventionalcommits are allowed? what if I want to use keep-a-changelog?

**Comment by edbzn** [comment]
@airtonix Yes, only angular and conventional preset from conventional changelog are allowed right now. The problem with Keep a Changelog is that we can't rely on semver to bump the packages, so I don't think it's doable easily with the current state.
